docs/schemas/itemMeta.json

```diff
@@ -478,6 +489,26 @@
         "title": {
           "$ref": "#/definitions/title"
         },
+        "subtitle": {
+          "$ref": "#/definitions/subtitle",
+          "description": "It's only usable when presentation of cascade item is showCards"
+        },
+        "description": {
+          "$ref": "#/definitions/description",
+          "description": "It's only usable when presentation of cascade item is showCards"
+        },
+        "footer": {
+          "$ref": "#/definitions/footer",
+          "description": "It's only usable when presentation of cascade item is showCards"
+        },
+        "scale": {
+          "type": "number",
+          "description": "Start from 0. Determine the scale of cascade choice. Scale is for determining the colour of card component. It's only usable when presentation of cascade item is showCards"
+        },
+        "isDisabled": {
+          "type": "boolean",
+          "description": "will disable choice"
```

Comment:
wow that's mighty powerful, it disables the users choice! Perhaps "disables this item"?

Reply:
will do. :P